### PR TITLE
Load migrations from all bundles that are shipped with the plugin

### PR DESCRIPTION
**Pull request by JoshuaBehrens**
### 1. Why is this change necessary?

When you add a bundle in the additional bundles, they shall be shopware bundles. Shopware bundles can have migrations. When I name dependent bundles, those should execute their migrations as well.

### 2. What does this change do, exactly?

I use the same strategy to load the additional bundles to get the additional migrations. As the migration source for a plugin is created at different places I could not change its content so it is not reliable to change.

### 3. Describe each step to reproduce the issue or behaviour.

1. Create a bundle with migrations
2. Refer to it in the additional bundles
3. Have code based on the migrations in the bundle
4. Your plugin bricked

### 4. Checklist

- [ ] I have written tests and verified that they fail without my change
- [ ] I have created a [changelog file](https://github.com/shopware/platform/blob/trunk/adr/2020-08-03-Implement-New-Changelog.md) with all necessary information about my changes
- [ ] I have written or adjusted the documentation according to my changes
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

### CODE REVIEW

File: src/Core/Framework/Plugin/PluginLifecycleService.php [hunk 1]
```diff
@@ -113,12 +117,13 @@ public function installPlugin(PluginEntity $plugin, Context $shopwareContext): I
         $pluginBaseClass = $this->getPluginBaseClass($plugin->getBaseClass());
         $pluginVersion = $plugin->getVersion();
 
+        $migrations = $this->createMigrationCollection($pluginBaseClass);
```
Comment (keulinho) [inline src/Core/Framework/Plugin/PluginLifecycleService.php:120]: Can you add a testcase for this change?

You can add it to `\Shopware\Core\Framework\Test\Plugin\PluginLifecycleServiceTest`, there are also fixture plugins, please add a plugin with own migrations and migrations in a bundle and check that both migrations are executed when the plugin is installed or updated.

And while your own it, please add a small changelog.

Comment (JoshuaBehrens) [reply src/Core/Framework/Plugin/PluginLifecycleService.php:120]: I can do that :) thanks for the clear instruction, where to dive into next.

**Comment by shyim** [comment]
I will close this pull request as its inactive. Feel free to create a new one updated

**Comment by JoshuaBehrens** [comment]
I did https://github.com/shopware/platform/pull/2583

**Pull request closed without merging**